recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
Should add that personally I would also be ok with just including those files with the recipe. This seems like a special case recipe where that is totally appropriate.